src/main/java/edu/kis/powp/jobs2d/features/MacroFeature.java

```diff
@@ -0,0 +1,62 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.builder.CompoundCommandBuilder;
+import edu.kis.powp.jobs2d.events.SelectMacroOptionListener;
+
+public class MacroFeature {
+    private static Application application;
+    private static CompoundCommandBuilder recordedCommand;
+
+    private static boolean isRecording = false;
+
+    public static void setupMacroFeature(Application app) {
+        recordedCommand = new CompoundCommandBuilder();
+        recordedCommand.setName("Record command");
+        application = app;
+
+        SelectMacroOptionListener clearOption = new SelectMacroOptionListener(0);
+        SelectMacroOptionListener startOption = new SelectMacroOptionListener(1);
+        SelectMacroOptionListener runOption = new SelectMacroOptionListener(2);
+
+        application.addComponentMenu(MacroFeature.class, "Macro");
+        application.addComponentMenuElement(MacroFeature.class, "Clear", clearOption);
+        application.addComponentMenuElementWithCheckBox(MacroFeature.class, "Start/Stop", startOption, false);
+        application.addComponentMenuElement(MacroFeature.class, "Run", runOption);
```

Comment:
Macro should not run commands. Command should be loaded using DriverCommandsManager (setCurrentCommand).

Reply:
OK, but change naming from RUN to LOAD